client/src/lit/play/console-utils.js

```diff
@@ -17,6 +18,17 @@ export function formatArray(input) {
       output += "Array [";
       output += formatArray(input[i]);
       output += "]";
+    } else if (!input.hasOwnProperty(i)) {
+      let emptyCount = 1;
+      while (i + 1 < l && !input.hasOwnProperty(i + 1)) {
+        emptyCount++;
+        i++;
+      }
+      if (emptyCount === 1) {
+        output += "<empty>";
```

Comment:
I would follow Firefox's formatting of `<1 empty slot>` here

Reply:
Fixed via b5ac744385.